charts/infra-apps/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
This needs to be reviewed and unused parts should be removed, the move to a library argoconfigs comes first though.